test/e2e/utils/test_context.go

```diff
@@ -161,6 +161,8 @@ func (kc *KBTestContext) Make(makeOptions ...string) error {
 
 // CleanupImage is for cleaning up the docker images for testing
 func (kc *KBTestContext) Destroy() {
+	// nolint:gosec
+	// G204: Subprocess launched with function call as argument or cmd arguments (gosec)
```

Comment:
We should keep `// nolint:gosec`
And drop `// G204: Subprocess launched with function call as argument or cmd arguments (gosec)`